### PR TITLE
Fix symlink command for bookmark sharing

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

File: cyberduck/bookmarks.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ This only works properly while using the standalone version of Cyberduck or Moun
 1. Install the Dropbox app, log into your Dropbox account and synchronize the folder where you want to have the bookmarks to the local disk.
 2. Quit *Cyberduck* and/or *Mountain Duck*.
 3. Navigate to `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/` and move the folders *Bookmarks* and if available *Profiles* to the desired location on Dropbox.
-4. Open *Terminal.app* and execute the command `ln -s <AppSupportDirectory/foldername> <CloudDirectory>` to create the symbolic link.
+4. Open *Terminal.app* and execute the command `ln -s <CloudDirectory> <AppSupportDirectory/foldername>` to create the symbolic link.
```
Comment (dkocher) [inline cyberduck/bookmarks.md:224]: Make the example specific like 

```
ln -s ~/Dropbox/Bookmarks ~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks
```

**Review by dkocher — CHANGES_REQUESTED** [review]

File: cyberduck/bookmarks.md (hunk 1 above)
Comment (AliveDevil) [reply cyberduck/bookmarks.md:224]: Wouldn't this then need an example for every single sync service?
Like Google Drive, OneDrive, Dropbox and iCloud?

Comment (dkocher) [reply cyberduck/bookmarks.md:224]: Specific examples are easier to understand and adapt.

File: cyberduck/bookmarks.md [hunk 2]
```diff
@@ -221,7 +221,8 @@ This only works properly while using the standalone version of Cyberduck or Moun
...
+e.g.: `ln -s ~/Dropbox/Bookmarks ~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks`
```
Comment (dkocher) [inline cyberduck/bookmarks.md:225]: Must escape white space
```suggestion
e.g.: `ln -s ~/Dropbox/Bookmarks ~/Library/Group Containers/G69SCX94XU.duck/Library/Application\ Support/duck/Bookmarks`
```

**Review by dkocher — CHANGES_REQUESTED** [review]

File: cyberduck/bookmarks.md [hunk 3]
```diff
@@ -221,7 +221,8 @@ This only works properly while using the standalone version of Cyberduck or Moun
 1. Install the Dropbox app, log into your Dropbox account and synchronize the folder where you want to have the bookmarks to the local disk.
 2. Quit *Cyberduck* and/or *Mountain Duck*.
 3. Navigate to `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/` and move the folders *Bookmarks* and if available *Profiles* to the desired location on Dropbox.
-4. Open *Terminal.app* and execute the command `ln -s <AppSupportDirectory/foldername> <CloudDirectory>` to create the symbolic link.
+4. Open *Terminal.app* and execute the command `ln -s <CloudDirectory> <AppSupportDirectory/foldername>` to create the symbolic link.
+e.g.: `ln -s ~/Dropbox/Bookmarks ~/Library/Group Containers/G69SCX94XU.duck/Library/Application\Support/duck/Bookmarks`
```
Comment (dkocher) [inline cyberduck/bookmarks.md:225]: Missing whitespace after `\`

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by dkocher — APPROVED** [review]

**Pull request merged**